### ISSUE TITLE
Gevent And Greenlet issue on python3.7

### ISSUE THREAD

**Issue by king-11** [HTTP-APIs/hydrus#532]
### I'm submitting a 
- [x] bug report.
- [ ] feature request.

### Current Behaviour:

gevent has binary compatibility issues with python3.7+

### Expected Behaviour:

It will run on python3.7 too without any issues [Reference](https://github.com/gevent/gevent/issues/1260)

### Steps to reproduce:

In python3.7 run
```zsh
pip3 install -r requirements.txt
python3 setup.py install
hydrus serve
```

### Do you want to work on this issue?

yes

**Comment by king-11** [comment]
Upgrading gevent to 20.9.0 and greenlet to 0.4.17 makes hydrus work in both py3.7 and 3.8

**Comment by king-11** [comment]
@Mec-iS  @sameshl any reviews on this 😅

**Comment by sameshl** [comment]
@king-11 You can go ahead and create a PR for this issue :+1:

**Comment by king-11** [comment]
closed by #537
